news/_posts/2021-05-05-community-survey-results.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Results of community survey – April 2021"
+subtitle: "Thank you for your feedback about the open-source project and the contributions"
+date: 2021-05-05 12:00:00
+authors: [MarionFrancois]
+icon: icon-world
+image: /assets/images/2021/05/build-survey-banner.png
+---
+
+The core product team carried out a survey in April, oriented around the open-source project and for the community. The goal of this survey was to better know and understand the pains and the wishes of the community about the open-source project.
+
+![Build survey banner](/assets/images/2021/05/build-survey-banner.png)
+
+## :two_hearts: Special thanks to the participants
+
+We received 98 responses, which makes a completion rate of 28,6%, and wanted to thank everyone who participated in this survey. Your opinions are very important for us and we care a lot about your satisfaction!
+
+## Results of the community survey of April 2021
+
+### How do you contribute to the open-source project?
+
+The answers to the first question show that the majority of contributors report bugs and suggest improvements or new features for the open-source project. 
+
+The code contribution with pull requests comes in 3rd position with 35,7%.
+
+And 21,4% of respondents do not contribute to the project.
+
+![How do you contribute to the open-source project?](/assets/images/2021/05/how-do-you-contribute.png)
+
+Remember that the best way to contribute to the open-source project is to open an issue when you are facing a problem and to fix it with a pull request if you have the technical skills. 
+If you are not a developer, you can consider hiring a specialist to help you. If you manage to get the problem fixed, then contribute the fix back to the project as a Pull Request. Remember that the more people contribute, the better PrestaShop becomes for everyone :muscle:
+
+### When do you contribute to the open-source project?
+
+54,5% of the respondents contribute during their working time, while 45,5% contribute in their free time.
```

Comment:
54.5% of the respondents contribute during their working time, while 45.5% contribute in their free time.